lib/matplotlib/lines.py

```diff
@@ -507,18 +507,27 @@ def set_pickradius(self, d):
 
     def get_fillstyle(self):
         """
-        return the marker fillstyle
+        Return the marker fillstyle.
```

Comment:
Do you want to also wrap these one-liners back to, well, a single line too?

Reply:
I'm considering to add ``See also `.set_fillstyle`` lines to the getters. That would undo the one-lining again.

I'm not quite decided yet if that's a good idea. So I left the one-liners for the moment. Any comment on linking to the setters? I wouldn't want to use a "See also" section as that would get too verbose.

Reply:
Seems fine.

Reply:
I will leave that for a separate PR. IMO keeping the PRs reasonably small and not mixing too many aspects/concepts makes them easier to review.